polkadot/runtime/kusama/src/weights/pallet_nomination_pools.rs

```diff
@@ -600,4 +600,8 @@ impl<T: frame_system::Config> pallet_nomination_pools::WeightInfo for WeightInfo
 			.saturating_add(T::DbWeight::get().reads(4))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+
+	fn adjust_ed_deposit() -> Weight {
+		Weight::default()
```

Comment:
Need to run the bench bot?

Reply:
Bench bot is not working anymore since kusama and polkadot runtime are not supposed to be used from Polkadot-sdk. I need to leave a placeholder for the code to compile but make sure this is benched in the fellowship runtime pr?